initializer/mount.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2024 Edgeless Systems GmbH
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package main
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	"github.com/edgelesssys/contrast/internal/logger"
+	"github.com/edgelesssys/contrast/internal/mount"
+	"github.com/spf13/cobra"
+)
+
+// luksVolume holds configuration for mounting a LUKS encrypted device.
+type luksVolume struct {
```

Comment:
```suggestion
type parsedCryptsetupFlags struct {
```

This is more in line with the naming in the user-facing CLI and also clearer in that this struct is thightly coupled with the CLI command and its flags.